### ISSUE TITLE
Segmentation fault error

### ISSUE THREAD

**Issue by lfdeep** [uber-research/UPSNet#59]
Hello!When i run train.py  in coco datasets, and i only have one gpu ,it occured:
> /UPSNet-master/upsnet/models/resnet_upsnet.py(125)forward()
-> fcn_output = self.fcn_head(*[fpn_p2, fpn_p3, fpn_p4, fpn_p5, fcn_rois])
(Pdb) n
Segmentation fault

Is the code wrong?

**Comment by lfdeep** [comment]
> hello, I have the same error with you, how did you deal with it? Thanks!

I have solved this problem and you can:
Breakpoint debugging, check if the error is in deform conv. If yes, it is because the environment is not configured, check whether the environment is compiled #6 .
it is recommended that you use gcc>4.9.

**Comment by AugustYuan** [comment]
My gcc version is 7.2.0, and I find the error as follows, so how to deal with it?

/home/kunyuan/anaconda3/lib/python3.6/site-packages/torch/nn/modules/module.py(477)__call__()
-> result = self.forward(*input, **kwargs)
(Pdb) n
/home/kunyuan/anaconda3/lib/python3.6/site-packages/torch/nn/functional.py:1006: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
Segmentation fault

**Comment by AugustYuan** [comment]
My gcc version is 7.2.0, and I find when the program runs to "output=train_model(*batch)" it will print error as follows, so how to deal with it?

/home/kunyuan/anaconda3/lib/python3.6/site-packages/torch/nn/modules/module.py(477)call()
-> result = self.forward(*input, **kwargs)
(Pdb) n
/home/kunyuan/anaconda3/lib/python3.6/site-packages/torch/nn/functional.py:1006: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
Segmentation fault

**Comment by lfdeep** [comment]
> My gcc version is 7.2.0, and I find when the program runs to "output=train_model(*batch)" it will print error as follows, so how to deal with it?
> 
> /home/kunyuan/anaconda3/lib/python3.6/site-packages/torch/nn/modules/module.py(477)call()
> -> result = self.forward(*input, **kwargs)
> (Pdb) n
> /home/kunyuan/anaconda3/lib/python3.6/site-packages/torch/nn/functional.py:1006: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
> warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
> Segmentation fault

cd upsnet/operators/_ext/deform_conv and run python
import torch and import deform_conv_cuda manually, it should be no problem if your environment setup is correct.

**Comment by AugustYuan** [comment]
I imported deform_conv_cuda successfully, but there still "segmentation fault" error.

**Comment by Flames60** [comment]
> I imported deform_conv_cuda successfully, but there still "segmentation fault" error.

Same error. Have you solved it?

**Comment by Flames60** [comment]
> I imported deform_conv_cuda successfully, but there still "segmentation fault" error.

I solved it by upgrading gcc from 4.8.5 to 4.9.4.

**Comment by AMzhanghan** [comment]
Same error. Have you solved it?

**Comment by AMzhanghan** [comment]
> I imported deform_conv_cuda successfully, but there still "segmentation fault" error.

Same error. Have you solved it?
